### PR TITLE
Add documentation on meeting best practices.

### PR DESCRIPTION
**Pull request by dlorenc**
This adds some best-practices around holding meetings, as discussed
in the governance meeting.

### CODE REVIEW

**Comment by dlorenc** [comment]
/hold

I'd like a few people to take a look at this.

File: process.md [hunk 1]
```diff
@@ -238,4 +238,26 @@ a project can get its own top level repo in [the `tektoncd` org](https://github.
...
+### Meetings
+
+Despite the best wishes of many engineers, meetings are sometimes necessary in the software development process.
+We expect that engineers will meet with each other from time to time to discuss designs, resolve issues and
+brainstorm ideas.
+
+There is no requirement that all meetings take place publicly.
+Face to face meetings between a small number of engineers, meetings internal to a single company, and ad-hoc
+discussions will always occur.
+Whenever you feel a meeting has touched on a topic of interest to the broader community, please make an effort
```
Comment (vdemeester) [inline process.md]: :100: I think we need to be less shy at sending emails to `tekton-dev@` :angel:

File: process.md [hunk 2]
```diff
@@ -238,4 +238,26 @@ a project can get its own top level repo in [the `tektoncd` org](https://github.
...
+to summarize this discussion in notes, an issue or an email.
```
Comment (bobcatfish) [inline process.md]: can link here to our mailing list

**Comment by bobcatfish** [comment]
Added a couple minor comments but otherwise

/lgtm

**Comment by bobcatfish** [comment]
Rebased in https://github.com/tektoncd/community/pull/61

**Pull request closed without merging**